demos/demo-list.js

```diff
@@ -1,9 +1,11 @@
 var demoList = [
   { name: "Simple Print", url: "simple-print.slang" },
   { name: "Simple Image", url: "simple-image.slang" },
+  { name: "Load From URL", url: "load-from-url.slang" },
```

Comment:
This can be confusing as it sounds like the option is for loading a shader from URL.
Rename to "Image at URL" ?